components/MetricsTable.tsx

```diff
@@ -93,7 +93,15 @@ const MetricDetail = ({ metricBare }: { metricBare: MetricBare }) => {
 /**
  * Renders a table of "bare" metric information.
  */
-const MetricsTable = ({ metrics }: { metrics: MetricBare[] }) => {
+const MetricsTable = ({
+  metrics,
+  canEditMetrics,
+  onEditMetric,
+}: {
+  metrics: MetricBare[]
+  canEditMetrics?: boolean
+  onEditMetric?: (metricId: number) => void
```

Comment:
Do we need the `canEditMetrics` argument? We can simply rely on `onEditMetric` being defined as an indication that an edit button is needed.

```suggestion
  onEditMetric,
}: {
  metrics: MetricBare[]
  onEditMetric?: (metricId: number) => void
```